o4/o4.py

```diff
@@ -1057,22 +1071,36 @@ def o4_status(changelist, depot):
         cur = None
     if cur is None:
         cur, fname = get_fstat_cache(changelist)
+    o4_log('fstat',
+           depot,
+           changelist=changelist,
+           cur=cur,
+           check_all=check_all,
+           quick=quick)
     if cur is None:
         print("*** ERROR: Current changelist could not be determined.")
         return
 
     print(f"Current changelist: {cur:,d}")
     print(f"  - HEAD is {changelist:,d} (+{changelist-cur:,d})")
 
+    os.environ['O4_PROGRESS'] = 'false'
     keep_case = f'| {o4bin} keep --case ' if sys.platform == 'darwin' else ''
-    cmd = f"{o4bin} fstat .@{cur} {keep_case}| {manibin} o4 drop --checksum"
+    drop_deleted = f"| grep -v ',$' " if not check_all else ''
+    changed = f" --changed {cur*4//5}" if quick else ''
+    cmd = f"{o4bin} fstat -v .@{cur}{changed}{drop_deleted}| {manibin} o4 drop --checksum{keep_case}"
```

Comment:
By putting keep_case check *after* the checksum test I cut the processing time by 40%. I wonder how many other places we could move it to after something?